web/js/waste.js

```diff
@@ -199,4 +199,17 @@ $(function() {
         updateTotal();
     });
 
+    if (fixmystreet.cobrand == 'brent') {
+        $('[id^="form-item_notes_"]').hide();
+        $('.govuk-select[name^="item_"]').change(function(e) {
+            var $this = $(this);
+            var $notes = $this.closest('.bulky-item-wrapper').find('[id^="form-item_notes_"]');
+            if ($this.val() == 'Small electricals: Other item under 30x30x30 cm') {
+                $notes.show();
+            } else {
+                $notes.hide();
+            }
+        });
+    }
+
```

Comment:
This isn't triggered on a page back/refresh

Reply:
I think changing to call the function on every one at load, and then same on change works:
```
         if (fixmystreet.cobrand == 'brent') {
-            $('[id^="form-item_notes_"]').hide();
-            $('.govuk-select[name^="item_"]').change(function(e) {
+            function update_small_items_other_notes() {
                 var $this = $(this);
                 var $notes = $this.closest('.bulky-item-wrapper').find('[id^="form-item_notes_"]');
                 if ($this.val() == 'Small electricals: Other item under 30x30x30 cm') {
@@ -171,7 +170,9 @@ $(function() {
                 } else {
                     $notes.hide();
                 }
-            });
+            }
+            $('.govuk-select[name^="item_"]').change(update_small_items_other_notes);
+            $('.govuk-select[name^="item_"]').each(update_small_items_other_notes);
         }
```